### DISCUSSION TITLE
List checkbox

### DISCUSSION THREAD

**Discussion by franklupo** [microsoft/fluentui-blazor#2295] [General]
Exist a list checkbox?

![image](https://github.com/microsoft/fluentui-blazor/assets/6276309/f78f071c-018c-42a9-b310-257060cfb1b5)

best reagrds

**Comment by vnbaaij** [comment]
No, we don't have that.

**Reply by dvoituron** [reply]
You can use a `FluentListbox` with a `OptionTemplate`.

**Reply by franklupo** [reply]
Multi select work?

![image](https://github.com/microsoft/fluentui-blazor/assets/6276309/32989925-0eac-4a37-b0a7-92fd8160d4a7)

**Reply by dvoituron** [reply]
No

**Reply by franklupo** [reply]
what is the problem?

**Reply by vnbaaij** [reply]
The listbox web component does not support multiple selection. It was built but never got merged into the FAST element/FAST foundation version the web components ar built on

**Reply by gotmoo** [reply]
Would it make sense to build something using FluentDataGrid, or is that too bulky?
![image](https://github.com/microsoft/fluentui-blazor/assets/25994839/a34ee555-c060-4de3-bb78-e8d54c05a0fa)

**Reply by vnbaaij** [reply]
> Would it make sense to build something using FluentDataGrid, or is that too bulky? 

Technically no issue, but maybee indeed a bit big/overkill from an UI perspective
